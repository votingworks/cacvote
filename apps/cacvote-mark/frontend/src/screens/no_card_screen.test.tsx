import { renderWithThemes as render } from '@votingworks/ui';
import { WelcomeScreen } from './welcome_screen';

test('renders', () => {
  const { container } = render(<WelcomeScreen />);
  expect(container).toMatchSnapshot();
});
