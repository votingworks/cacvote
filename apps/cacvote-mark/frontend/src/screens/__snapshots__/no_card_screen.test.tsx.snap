// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="sc-jlZhew hVbvRB"
  >
    <main
      class="sc-dZoequ gTzqFs"
    >
      <h1
        class="sc-dcJsrY cbAbxH"
      >
        Welcome
      </h1>
      <p
        class="sc-eqUAAy ePbBJT"
      >
        Insert your Common Access Card to begin.
      </p>
    </main>
  </div>
</div>
`;
